src/EspNowPreferences.h

```diff
@@ -34,6 +34,19 @@ class EspNowPreferences : public EspNowNetwork::Preferences {
    */
   bool espNowGetMacForHost(uint8_t *buffer) override;
 
+  /**
+   * @brief Set the WiFi channel that the host is on
+   */
+  bool espNowSetChannelForHost(uint8_t channel) override;
+
+  /**
+   * @brief Return the WiFi channel stored, or std::nullopt if no channel stored.
```

Comment:
```suggestion
   * @brief Get the WiFi channel stored. Returns false if no channel stored.
```
(I know you copy-pasted this from above, which is also wrong. I copy-pasted this code form another project where I use std::optional as an return argument instead of having a bool return and pointer as arg)

Reply:
fixed